tests/driver/cpp/cppdriver.rs

```diff
@@ -111,6 +111,11 @@ pub fn test(testcase: &test_driver_lib::TestCase) -> Result<(), Box<dyn Error>>
         cmd = std::process::Command::new(binary_path.deref());
     }
 
+    #[cfg(target_os = "unix")]
+    cmd = cmd.env("LD_LIBRARY_PATH", env!("CPP_LIB_PATH"));
+    #[cfg(target_os = "dawing")]
```

Comment:
Is this correct?

Reply:
No, it's wrong - as well as the whole syntax. That was a late night attempt at doing something I should do today with a fresh mind and mug of coffee :-)

Reply:
Been there, done that:-) Managed to get my C++ tests to run again at about 1am today;-)